### PR TITLE
Soarian FHIR R4 Updates For Certification

### PR DESCRIPTION
**Pull request by VamsiMure**
Description
----

PR Checklist
----
- [ ] Screenshot(s) of changes attached before changes merged.
capability statement issue fix
capability statement issue fix(2)
capability statement issue fix (3)
capability statement issue fix(4)
- [ ] Screenshot(s) of changes attached after changes merged and published.

### CODE REVIEW

**Review by PBKarthik — APPROVED** [review]

**Review by stanlysampath — APPROVED** [review]

**Review by PBKarthik — APPROVED** [review]

**Review by PBKarthik — APPROVED** [review]

File: content/soarian/r4/clinical/diagnostics/diagnostic-report.md [hunk 1]
```diff
@@ -26,6 +26,8 @@ The search results include the following fields if they contain values:
 	* [Effective period](https://hl7.org/fhir/r4/diagnosticreport-definitions.html#DiagnosticReport.effective_period){:target="_blank"}
 * [Issued](https://hl7.org/fhir/r4/diagnosticreport-definitions.html#DiagnosticReport.issued){:target="_blank"}
 * [Performer](https://hl7.org/fhir/r4/diagnosticreport-definitions.html#DiagnosticReport.performer){:target="_blank"}
+	* Practitioner
+    * Organization
```
Comment (ashu-bdvt) [inline content/soarian/r4/clinical/diagnostics/diagnostic-report.md:30]: Indentation mismatch

**Review by ashu-bdvt — CHANGES_REQUESTED** [review]
Also please add 2-3 more screenshots highlighting the changes to the menus and other documentation changes

File: content/soarian/r4.md [hunk 2]
```diff
@@ -375,7 +375,9 @@ Certain data may not be available to all applications or users. Encounter or org
...
..., patient/CareTeam.read, patient/Condition.read, patient/Device.read, patient/DiagnosticReport.read, patient/DocumentReference.read, patient/Encounter.read, patient/Goal.read, patient/Immunization.read, patient/MedicationRequest.read, patient/Observation.read, patient/Patient.read, patient/Procedure.read, patient/Provenance.read, user/AllergyIntolerance.read, user/Binary.read, user/CarePlan.read, user/CareTeam.read, user/Condition.read, user/Device.read, user/DiagnosticReport.read, user/DocumentReference.read, user/Encounter.read, user/Goal.read, user/Immunization.read, user/MedicationRequest.read, user/Observation.read, user/Organization.read, user/Patient.read, user/Person.read, user/Practitioner.read, user/Procedure.read, user/Provenance.read,   system/AllergyIntolerance.read, system/Binary.read, system/CarePlan.read, system/CareTeam.read, system/Condition.read, system/Device.read, system/DiagnosticReport.read, system/DocumentReference.read, system/Encounter.read, system/Goal.read, system/Immunization.read, system/MedicationRequest.read, system/Observation.read, system/Organization.read, system/Patient.read, system/Practitioner.read, system/Procedure.read, system/Provenance.read
```
[the remark this answers is not in the document]
Comment (VamsiMure) [reply content/soarian/r4.md:380]: Could you be more specific on whats the change here. Is it the space ?

[the remark this answers is not in the document]
Comment (VamsiMure) [reply content/soarian/r4.md]: I have updated the content as suggested

File: content/soarian/r4/clinical/diagnostics/diagnostic-report.md (hunk 1 above)
Comment (VamsiMure) [reply content/soarian/r4/clinical/diagnostics/diagnostic-report.md]: updated

File: content/soarian/r4.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (ashu-bdvt) [reply content/soarian/r4.md]: > Could you be more specific on whats the change here. Is it the space ?

yes

**Review by ashu-bdvt — APPROVED** [review]

**Comment by aki-mathur** [comment]
Screenshot 2022-08-04 at 7 03 28 PM

Screenshot 2022-08-04 at 7 03 54 PM

**Pull request merged**